### ISSUE TITLE
storage-cmdline-sample

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/google-api-java-client#777]
```
Which Google API and version (e.g. Google Calendar API version 3)?
Google Cloud Storage JSON API v1beta2

Java environment (e.g. Java 6, Android 2.3, App Engine)?
Java 6.

External references, such as API reference guide?
https://developers.google.com/storage/docs/json_api/v1

```

Original issue reported on code.google.com by `nherr...@google.com` on 9 May 2013 at 10:46

**Comment by GoogleCodeExporter** [comment]
Original comment by `nherr...@google.com` on 10 May 2013 at 9:43

**Comment by GoogleCodeExporter** [comment]
```
Fixed in commit 3c0563b18af3.
```

Original comment by `nherr...@google.com` on 13 May 2013 at 8:10
- Changed state: **Done**
